src/database/CMakeLists.txt

```diff
@@ -1,5 +1,29 @@
-add_library(example-lib Example.cpp)
+cmake_minimum_required(VERSION 3.12)
```

Comment:
cmake_minimum version is set to 3.20 in the main CMakeLists. It will be better to keep one version.